frontend/apps/reader/modules/readerlink.lua

```diff
@@ -22,7 +22,8 @@ local Screen = Device.screen
 local T = ffiutil.template
 
 local ReaderLink = InputContainer:extend{
-    location_stack = nil, -- table, per-instance
+    location_stack = nil, -- table, per-instance,
+    alt_handlers_buttons = {}
```

Comment:
Should be per-instance, in :init().

Reply:
This is still an issue in the latest version ;).

Reply:
And you'll want to possibly make it clearer whether it should be provided by the constructor table, or, more likely given the latest changes, at runtime via the added API ;).